pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel", "numpy"]
+requires = ["setuptools", "setuptools_scm", "wheel", "numpy"]
```

Comment:
We should pin numpy to the minimum version here: `numpy==1.13.0`

Reply:
Done.